src/stubs/languages/qml/MuseScore/Languages/LanguagesPage.qml

```diff
@@ -0,0 +1,6 @@
+import QtQuick 2.15
+
+Item {
+    property string search: ""
+    property string backgroundColor: ""
+}
```

Comment:
I think it better to add Text like "Languages Page Stub"

Reply:
added